extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesConfigSourceProvider.java

```diff
@@ -38,10 +41,10 @@ public KubernetesConfigSourceProvider(KubernetesConfigSourceConfig config, Kuber
         }
 
         List<ConfigSource> result = new ArrayList<>();
-        if (config.configMaps.isPresent()) {
+        if (config.enabled && config.configMaps.isPresent()) {
```

Comment:
>enabling secrets, then config maps are enabled

I believe I've modified the configMap to not load the configMap if `quarkus.kubernetes-config.enabled=false`. What do you think?

Reply:
I'll take a closer look tomorrow

Reply:
Ah right, I hadn't seen this!